bumble/hci.py

```diff
@@ -1995,6 +2004,19 @@ def __str__(self):
         return self.to_string()
 
 
+# -----------------------------------------------------------------------------
+@dataclasses.dataclass
+class HCI_Dataclass_Object(HCI_Object):
+    def __post_init__(self) -> None:
+        self.fields = HCI_Object.fields_from_dataclass(self)
+
+    @classmethod
+    def parse_from(cls, data: bytes, offset: int) -> tuple[int, Self]:
```

Comment:
I suggest calling this `parse_from_bytes` to be consistent with similar methods elsewhere.

Reply:
Done